src/ui/Progress/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Circle } from "rc-progress";
+import styles from "./styles.module.css";
+
+export interface KudosProgressProps {
+  percent: number;
+  currentKudos?: number;
```

Comment:
Should current kudos not always exist? If there arent kudos, it should be 0, right?